src/components/experiment/ExperimentItem/ExperimentItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Button from 'react-bootstrap/Button';
+
+interface ExperimentItemProps {
+  url: string
+}
+
+async function deleteExperimentFile(filename: string) {
+  await fetch(`${process.env.REACT_APP_BACKEND_ADDRESS}/experiment/${filename}`, { method: 'delete'});
```

Comment:
What if the delete call fails?

Reply:
It will display the message returned from the delete response